microsoft-edge/extensions-chromium/publish/user-reviews.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Respond to user reviews about your extension
+description: Developers can view their Microsoft Edge extension’s reviews and ratings and respond to user reviews from the Partner Center dashboard.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 05/23/2021
+---
+# Respond to user reviews about your Microsoft Edge extension
+
+
+<!-- ====================================================================== -->
+Microsoft recently launched a new feature enabling you to respond to user reviews received on the Microsoft Edge Add-ons product page. For more information, see the blog [Microsoft Edge Add-ons | NEW Feature to manage user reviews on your extension](https://techcommunity.microsoft.com/t5/articles/microsoft-edge-add-ons-new-feature-to-manage-user-reviews-on/m-p/3372537).
```

Comment:
Why should we have the blog link right at the top of this article? I think it will be distracting and it will cause users to jump away from this page. Ideally we dont need this link. If it is recommended for SEO, have it at the end of the article.